common/crypto_utils/avalon_crypto_utils/crypto/crypto.i

```diff
@@ -18,7 +18,6 @@
 %{
```

Comment:
We don't need SWIG in avalon_crypto_utils. Only verify report need SWIG. So you can delete this file

Reply:
Ok, then crypto folder itself can be removed.

Reply:
Removing swig file needs lot of changes in listener and other files. We have to provide python alternatives for byte_array_to_str, random_bit_string etc.. To keep this PR simple, i will take this change in a separate pr.

Reply:
Let's add the required functions first (in a new PR). We can put this PR on hold till then. I don't see a point why we need to merge this PR till we avoid openSSL dependency completely in crypto utils.

Reply:
I created a new commit in the same PR to provide python definitions for common/cpp utils as alternate.